recipes/orca-test/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "orca_test" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/UDST/orca_test/archive/v{{ version }}.tar.gz
+  sha256: 3107b31578b2f6e45c5e57b8bc45c94bf2d59f3d3ad1d7162bf36537f7209cca
+
+build:
+  number: 0
```

Comment:
I think this can be `noarch: python`

https://conda-forge.org/docs/meta.html#building-noarch-packages